plotting/operationalPlots.py

```diff
@@ -856,28 +833,25 @@ def obs_V_mod_bathy(ofname, p_dict, obs_dict, logo_path='ArchiveFolder/CHL_logo.
     # get the time strings!!
     obs_date = p_dict['obs_time'].strftime('%Y-%m-%d %H:%M')
     model_date = p_dict['model_time'].strftime('%Y-%m-%d %H:%M')
-    b, = ax1.plot(p_dict['x'], p_dict['obs'], 'r-', label='Observed (initial) \n' + obs_date)
+    b, = ax1.plot(p_dict['x'], p_dict['obs'], 'r-', label='Observed \n' + obs_date)
     c, = ax1.plot(p_dict['x'], p_dict['model'], 'y-', label='Model \n' + model_date)
-    if p_dict['obs2_time'] != []:
-        obs2_date = p_dict['obs2_time'].strftime('%Y-%m-%d %H:%M')
-        r, = ax1.plot(p_dict['x'], p_dict['obs2'], 'r--', label='Observed (final) \n' + obs2_date)
 
     # add altimeter data!!
     temp05 = Alt05['zb'][Alt05['plot_ind'] == 1]
     temp04 = Alt04['zb'][Alt04['plot_ind'] == 1]
     temp03 = Alt03['zb'][Alt03['plot_ind'] == 1]
     # Alt05
     f, = ax1.plot(Alt05['xFRF'] * np.ones(2), [temp05 - np.std(Alt05['zb']), temp05 + np.std(Alt05['zb'])], 'k-',
-                  label='Gauge Data')
-    g, = ax1.plot(Alt05['xFRF'] * np.ones(1), temp05, 'k_', label='Gauge Data')
+                  label='Gage Data')
```

Comment:
these should read Gauge

Reply:
I fixed this in all plot labels, print statements, and comments that I found, but I did **not** change it in the variable names or in the foldernames where the files are saved/copied (the one I found that has it in the name is - '/mnt/gaia/gages/results/frfIn/CMTB')